hack/scripts/configure_private_cluster.sh

```diff
@@ -0,0 +1,83 @@
+# Add the SQS and SSM VPC endpoints if we are creating a private cluster
```

Comment:
Can is it possible for us to tag all the we create resources with `testing/cluster`? We use that tag to be able to find resources to delete with our sweeper